src/main/resources/assets/twilightforest/lang/en_us.json

```diff
@@ -426,7 +411,6 @@
   "structure.final_castle": "Final Castle",
 
   // Items
-  "item.twilightforest.boarkchop": "Boarkchop",
```

Comment:
Almost ready to merge!

Boarkchop is dropped by the Boar's loottable, as a terrible pun 😄 

If you could please re-add it, then we can merge this PR! :)